EnglishFilter/sections/foreign.txt

```diff
@@ -792,6 +792,17 @@ kliuki.bg###container > div[style="float:left;"] > div[style]
 ! https://github.com/AdguardTeam/AdguardFilters/issues/33601
 lsj2017.com##.wrap > .top_box
 ||107.175.181.58^$domain=lsj2017.com
+! https://github.com/AdguardTeam/AdguardFilters/issues/33587
+||a304.top^$image,domain=hihbt.com
+||z2x5c8.com^$image,domain=hihbt.com
+hihbt.com###advert_widget
+hihbt.com##aside[id^="advert-"]
+hihbt.com##a[href*="18sapp.com"]
+hihbt.com##a[href*="twseb20.com"]
+hihbt.com##a[href*="ap996.co/"]
+hihbt.com##a[href*="baa.im/"]
```

Comment:
.textwidget  > p a[target="_blank"][rel="noopener"]:not([href^="/"]). Please also exchange search parameter for ext-CSS rules.